lib/src/utils/schema/actions/reducerActions.ts

```diff
@@ -21,7 +21,12 @@ export function reduceSchema(schema: VernaSchemaType, action: ReducerAction) {
     case VernaActionsEnum.ADD_SECTION:
       return addSection(schema, action.payload.id!, action.payload.SectionTemplate);
     case VernaActionsEnum.ADD_WIDGET:
-      return addWidget(schema, action.payload.id!, action.payload.widgetInfos!);
+      return addWidget(
+        schema,
+        action.payload.id!,
+        action.payload.widgetInfos!,
+        action.meta.formatMessage,
+      );
```

Comment:
I did not see code change to bind `formatMessage` within meta action in useSchemaReducer.

Reply:
Arg, i forgot to add it in the useSchemaReducer indeed, thanks